scripts/common

```diff
@@ -1,6 +1,24 @@
 #!/usr/bin/env bash
-set -ex
+set -exu
 
 # Export env vars needed for running test dependencies
 export ORG=mrcide
-export OUTPACK_SERVER_TAG=$ORG/outpack_server:main
+export OUTPACK_SERVER_TAG=$ORG/outpack_server:497d8b1
+
+export CONTAINER_NAMESPACE=orderly.runner
+
+export NETWORK=$CONTAINER_NAMESPACE-network
+export REDIS=$CONTAINER_NAMESPACE-redis
```

Comment:
this isnt really common right its specific to runner?...as per comment on `run-dependencies` this can be moved to script that spins up orderly.runner stuff....

Reply:
similar to above i would say its nice to have one place for these scripts, like i can just look here and see this is all the names of the containers or something like that, its not a long script, i think whitespace could be a sufficient separator for these concepts, i also would vote for just moving the db/scripts/run into here, it seems annoying to navigate 3 folder levels away to find a single script that just does a docker run, i think we would be at a risk of over abstracting where its not needed